roles/fio/files/runfio.sh

```diff
@@ -48,17 +74,17 @@ FIO_CMD="fio"
 FIO_ARGS="--group_reporting --direct=1 --ioengine=libaio"
 
 if [[ $timetorun -gt 0 ]]; then
-	FIO_ARGS="$FIO_ARGS --time_based --runtime=$timetorun"
+		FIO_ARGS="$FIO_ARGS --time_based --runtime=$timetorun"
```

Comment:
let's keep 4 spaces and have consistence with the indentation.

Reply:
This is fixed now. Also made 2 commits in this PR